package.json

```
@@ -6,12 +6,16 @@
   "scripts": {
     "build": "rm -rf artifacts & hardhat compile",
     "deploy": "hardhat run --network localhost scripts/deploy.js",
+    "erc20_e2e": "hardhat run --network localhost scripts/e2e.js",
     "eth_e2e": "hardhat run --network localhost scripts/ethEndToEndTest.js",
     "deposit": "hardhat run --network localhost scripts/deposit.js",
     "withdraw": "hardhat run --network localhost scripts/withdraw.js",
     "setAssetMeta": "node scripts/setAssetMeta.js",
     "setAssetMetaViaJSON": "node scripts/setMetadataViaStorage.js",
     "startEthNode": "./scripts/startEthNode.sh",
+    "validatorRelayer_local": "hardhat run --network localhost scripts/subscribeEventProof.js",
+    "validatorRelayer_ropsten": "hardhat run --network ropsten scripts/subscribeEventProof.js",
+    "validatorRelayer_mainnet": "hardhat run --network mainnet scripts/subscribeEventProof.js",
```

hmm I think can do:
```suggestion
    "validatorRelayer": "hardhat run --network $1 scripts/subscribeEventProof.js"
```

can run like:
`yarn validatorRelayer ropsten`

---

doesn't work this way... will try to investigate..